db/migrate/20240308005744_make_user_id_nullable_in_payments.rb

```diff
@@ -0,0 +1,7 @@
+# db/migrate/20240308005744_make_user_id_nullable_in_payments.rb
+
+class MakeUserIdNullableInPayments < ActiveRecord::Migration[7.1]
+  def change
+    change_column :payments, :user_id, :integer, null: true
```

Comment:
My preference is to create the reference like this including and index:
`add_reference :payments, :user, foreign_key: true, null: true, index: true`

To change this you need to run locally:
```
rails db:rollback
# change the contents of the migration from change_column to add_reference as listed about, then run the migration below
rails db:migrate
```

Reply:
Ok, so I had a few hurdles trying to get this to work. It seems to be working now (as I haven't received an error message 🤣). Would you mind checking this for me to ensure I have done it correctly please? @anandjacobmable @JosephA91 Thanks 🫶🏼